osmnx/pois.py

```diff
@@ -0,0 +1,414 @@
+################################################################################
+# Module: pois.py
+# Description: Download and plot Points of Interests (POIs) from OpenStreetMap
+# License: MIT, see full license in LICENSE.txt
+# Web: https://github.com/gboeing/osmnx
+################################################################################
+
+from shapely.geometry import Point, LineString, MultiPolygon, Polygon, box
+import geopandas as gpd
+from .core import overpass_request, bbox_from_point, gdf_from_place
+from .utils import log, geocode, bbox_to_poly
+
+def parse_poi_query(west, south, east, north, amenities=None, timeout=180, maxsize=''):
```

Comment:
This is probably the ideal argument order, but everywhere else in OSMnx, bounding-box extents are passed as north, south, east, west. For the sake of consistency, could you tweak the argument order as such?

Reply:
Done.